test/integration/fetch-collection/index.ts

```diff
@@ -292,6 +292,35 @@ describe("Fetching Collection", () => {
         });
     });
   });
+
+  describe("Fetching with multiple includes", () => {
+    it("should return organisation and schools related that people relate to", () => {
+      return Agent.request("GET", "/people")
+        .query("include=manages,homeSchool")
+        .accept("application/vnd.api+json")
+        .then(res => {
+          const stateGovernment =
+            res.body.included.filter(it => it.attributes.name === 'STATE GOVERNMENT');
+          const elementarySchool =
+            res.body.included.filter(it => it.attributes.name === 'ELEMENTARY SCHOOL');
+
+          expect(stateGovernment).to.have.length(1);
+          expect(elementarySchool).to.have.length(1);
+        });
+    });
+
+    it("should error if relationship names are encoded comma seperated", () => {
+      return Agent.request("GET", "/people")
+        .query("include=manages%2ChomeSchool")
+        .accept("application/vnd.api+json")
+        .then(() => {
```

Comment:
The linter [is complaining](https://circleci.com/gh/ethanresnick/json-api/625?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) about the empty function here. And, in this case, the empty function is actually problematic, because it means that the test will still pass even if the server doesn't error. You should `throw` inside this function so that the test fails if the request responses with a success status code.